lib/Optimizer/CodeGen/LowerToQIR.cpp

```diff
@@ -507,13 +549,12 @@ class OneTargetOneParamRewrite : public ConvertOpToLLVMPattern<OP> {
     auto loc = instOp.getLoc();
     ModuleOp parentModule = instOp->template getParentOfType<ModuleOp>();
     auto context = parentModule->getContext();
-    auto qirFunctionName = std::string(cudaq::opt::QIRQISPrefix) + instName;
+    auto qirQisPrefix = std::string(cudaq::opt::QIRQISPrefix);
```

Comment:
I think @schweitzpgi has been encouraging the use of `llvm::StringRef` and `llvm::Twine` in the compiler. Can `llvm::Twine` be used here?

Reply:
```suggestion
    std::string qirQisPrefix = cudaq::opt::QIRQISPrefix;
```
`llvm::Twine` is a foot gun to only be brought out of the box on very special holidays ... more specifically, it is _sometimes_ ok to use it to pass in a single argument that is capable of assembling a string in the called function because none of the parts to be assembled have vanished yet.